test/performance/README.md

```diff
@@ -49,6 +49,12 @@ if err != nil {
 }
 ```
 
+## Zipkin trace
+
+When `zipkinReqd` flag is enabled in the performance test setup, it turns on zipkin logging for all test requests
```

Comment:
I think we need to reword this to be more clear.  Something like:
Zipkin tracing can be enabled if needed by the perf test during setup. Once it is setup, all requests made by the `spoofingClient` will log json traces for responses with HTTP server errors(response codes between 500 and 600).

Sample trace:
...

Reply:
Thanks for the suggestion. Updated the README with the suggestion and included a partial trace for a 500 response.